src/components/file-browser/FileDetails.ts

```diff
@@ -0,0 +1,32 @@
+import filesize from 'filesize'
+import { dd, div, dl, dt } from 'react-hyperscript-helpers'
+import { FileBrowserFile } from 'src/libs/ajax/file-browser-providers/FileBrowserProvider'
+
+
+interface FileDetailsProps {
+  file: FileBrowserFile
+}
+
+export const FileDetails = (props: FileDetailsProps) => {
+  const { file } = props
+
+  return div({
+    style: {
+    }
```

Comment:
nit: can this empty style object be omitted?

Reply:
+1 and with that the outer `div` (starting after `return`) can be turned into an `h(Fragment, [...])` to avoid a redundant DOM element?